lib/salvator.js

```diff
@@ -84,8 +85,10 @@ module.exports = function(filepath, options) {
                     return next(new Error("Limit exceeded"));
                 }
 
-                fs.exists(filepath, function(exists) {
-                    next(null, exists);
+                fs.stat(filepath, function (err) {
```

Comment:
May be we could check the `filepath` to existing in `excluded` list before getting stats from `fs.stat`?
Like so:

```js
if (exclude.indexOf(filepath) !== -1) {
    return next(null, true);
}

// fs.stat(...
```

Or did I missed some logic? =)